Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+        "/me/hands/right": "semantic/controller/right"
+    }
+}
+````
+
+Take the time to review the device descriptor to understand how the Vive devices are factored into OSVR interfaces. The parts related to the puck are the assignment to tracker 3, and the automatic alias:
+````json
+"/me/puck": "semantic/puck",
+````
+
+### OSVR Server Configuration
+Normally for HDK2 with orientation-only tracking, we’d use a config file such as [/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json](https://github.com/OSVR/OSVR-Core/blob/master/apps/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json). 
+
+Take a look at the last line in the config file above. The syntax is similar to how we’ll use the tracking puck:
+````json
+"aliases": {"/me/head":"/com_osvr_Multiserver/OSVRHackerDevKit0/semantic/hmd"}
+````
+Instead of setting up an alias to the Multiserver plugin, we want to access the Vive plugin. We saw earlier that there is an automatic alias in the Vive device descriptor named “/me/puck”. Our new config file with puck tracking for the head is:
+````json
+"aliases": {"/me/head":"/me/puck”}
+````
+That’s it! We are now able to move the puck around to change the image in our display. We probably have a little extra work to do depending on the puck's expected orientation and how it is mounted to the HDK.
+
+### Rotating and Translating
+Depending on how your puck is mounted, there may be more configuration needed for tracking to appear in the correct orientation. We decided to mount the puck on top of HDK2 with the crown pointing up and microUSB port facing the forehead, like so:
+![Puck Mounted HDK2](https://github.com/OSVR/OSVR-Docs/blob/master/Configuring/images/hdk_puck.jpg)
+Doing so requires a -90 degree rotation, which is applied in the server config:
+````json
+"aliases": {
+	"/me/head": {
+      "rotate": {
+        "axis": "x",
+        "degrees": -90
+      },
+      "child": {
+        "changeBasis": {
+          "x": "x",
+          "y": "y",
+          "z": "z"
+        },
+        "child": "/me/puck"
+      }
+    }
+}
+````
+Note: the changeBasis isn't changing anything in this example, but it's left in this example anyway.
+
+Finally, we want to translate the origin of the tracker so that it's position appears to be at the center of the eyes. It's mounted a few inches above and in-front of the eyes, so we'll need to translate it towards the head and down in the server config file:
+````json
+"aliases": {
+	"/me/head": {
+      "posttranslate": [0.0, -0.1143, -0.0635],
+      "rotate": {
+        "axis": "x",
+        "degrees": -90
+      },
+      "child": {
+        "changeBasis": {
+          "x": "x",
+          "y": "y",
+          "z": "z"
+        },
+        "child": "/me/puck"
+      }
+    }
+}
+````
+Note: The posttranslate values here may need some fine-tuning depending on exact location of the tracking sensor in relation to the center of the eyes. The units are meters.
```

Comment:
You might also explain (or link to an explanation of) the coordinate system used by OSVR (_i.e._, +z is backward, &minus;z is forward, +x is to the right, &minus;x is to the left, +y is up, and &minus;y is down).